src/category_theory/abelian/basic.lean

```diff
@@ -386,6 +387,27 @@ is_image.iso_ext (coimage_strong_epi_mono_factorisation f).to_mono_is_image
 abbreviation image_iso_image : abelian.image f ≅ image f :=
 is_image.iso_ext (image_strong_epi_mono_factorisation f).to_mono_is_image (image.is_image f)
 
+namespace preserves_image
+
+variables {𝓐 𝓑 : Type u} [category.{v} 𝓐] [category.{v} 𝓑]
```

Comment:
Please correct me if this has changed, but I think mathlib style is to use normal letters for names of categories (cf. the rest of the two files you modified).

Reply:
done